lib/components/explorer_result_page_tick/explorer_result_page_tick.dart

```diff
@@ -88,28 +86,39 @@ class ExplorerResultPageTick extends StatelessWidget {
 
   @override
   Widget build(BuildContext context) {
-    return Column(
-      crossAxisAlignment: CrossAxisAlignment.start,
-      children: [
-        ExplorerResultPageTickHeader(
-            tickInfo: tickInfo,
-            onTickChange: onRequestViewChange != null
-                ? (tick) => {
-                      onRequestViewChange!(
-                          RequestViewChangeType.tick, tick, null)
-                    }
-                : null),
-        Padding(
-            padding: EdgeInsets.only(
-                left: ThemeEdgeInsets.pageInsets.left,
-                right: ThemeEdgeInsets.pageInsets.right),
-            child:
-                Column(crossAxisAlignment: CrossAxisAlignment.start, children: [
-              getTransactionsHeader(context),
-              tickInfo.transactions != null && tickInfo.transactions!.isNotEmpty
-                  ? listTransactions()
-                  : Container()
-            ])),
+    return CustomScrollView(
+      slivers: [
+        SliverToBoxAdapter(
+          child: ExplorerResultPageTickHeader(
+              tickInfo: tickInfo,
+              onTickChange: onRequestViewChange != null
+                  ? (tick) => {
+                        onRequestViewChange!(
+                            RequestViewChangeType.tick, tick, null)
+                      }
+                  : null),
+        ),
+        SliverPadding(
+          padding: EdgeInsets.only(
+              left: ThemeEdgeInsets.pageInsets.left,
+              right: ThemeEdgeInsets.pageInsets.right),
+          sliver: SliverToBoxAdapter(
+            child: getTransactionsHeader(context),
+          ),
+        ),
+        if (tickInfo.transactions != null && tickInfo.transactions!.isNotEmpty)
+          listTransactions(),
+        // Padding(
```

Comment:
if we won't use the commented code, please remove

Reply:
Done